ws/DiscordWebsocket.ts

```diff
@@ -158,15 +155,23 @@ export default class DiscordWebsocket {
   // TODO Construct proper errors
   private _onClose(code: GATEWAY_CLOSE_EVENT_CODES) {
```

Comment:
Disable event listener here

Reply:
Doesn't it stop listening when the connexion is closed?

Reply:
We get duplicate event listeners if we don't stop listening

Reply:
I disabled the `close` listener